### DISCUSSION TITLE
[solved] Please create a meta repository under tox-dev

### DISCUSSION THREAD

**Discussion by ssbarnea** [tox-dev/tox#1877] [Ideas]
Following practices from other similar organizations, please create a repository named "meta" under "tox-dev" organization, so we can use it to track things that are not specific to tox. That is especially useful for plugin authors and maintainers.

It would not be a bad idea to also pin this new meta repository in order to make it more visible to others when they visit organization main page.

For example, i would like a new `tox-bindep` repository for the basic plugin I am writing. 

Examples:
* https://github.com/pytest-dev/meta/issues 
* https://github.com/PyCQA/meta/issues
* https://github.com/sphinx-contrib/github-administration/issues

**Comment by gaborbernat** [comment]
Done https://github.com/tox-dev/meta and https://github.com/tox-dev
